### PR TITLE
add set path or secure interface

### PR DESCRIPTION
**Pull request by shoota**
set path and secure attributes

### CODE REVIEW

File: example/path/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```
Comment (shoota) [inline example/path/index.html]: `path` can set when access to url under path value.
ex) when access to `http://domain/path/`,  can set `path=/path`. if not, will ignore set cookie.

File: src/index.js [hunk 2]
```diff
@@ -6,12 +6,26 @@ function removeNaked (): string {
...
+  path: ?string,
+  secure: ?boolean
+): string {
   const value = cookies.get(name)
   if (value) {
     return value
   }
 
+  let attribute = { expires, secure }
+  if (path) {
```
Comment (shoota) [inline src/index.js]: js-cookie set to `/` as default, but if set `undefined` to path attribute, use access path. (For example, `/path`).

File: test/index.test.js [hunk 3]
```diff
@@ -27,9 +27,30 @@ describe('auto-cookie', () => {
...
+      .url('http://0.0.0.0:8000')
+      .getCookie(name)
+      .then(cookie => {
+        assert(!cookie)
+        done()
+      })
+  })
+})
+
+describe('path-cookie', () => {
```
Comment (9renpoto) [inline test/index.test.js]: please 1 file single describe, because `.only` options is effective range over callbacks

File: src/index.js [hunk 4]
```diff
@@ -20,28 +30,34 @@ function findOrCreate (name: string, expires: number, data: ?string): string {
...
+    cookies.set(name, data, attr)
   }
-  return cookies.get(name, { domain: location.hostname, expires })
+  return cookies.get(
+    name,
+    Object.assign({}, attr, { domain: location.hostname })
+  )
 }
 
 export function find (name: string, expires: number): string {
```
Comment (9renpoto) [inline src/index.js]: `find( name: string, options: Attributes): string`

**Pull request merged**